Cargo.toml

```diff
@@ -1,18 +1,19 @@
 [package]
 name = "polynomials"
-version = "0.2.4"
-authors = ["cargodog <cargodog@protonmail.com>"]
+version = "0.2.5"
+authors = ["cargodog <cargodog@protonmail.com>", "Drew Stone <drew@commonwealth.im>"]
 edition = "2018"
 description = "Tools to perform polynomial arithmetic and evaluation."
 license-file = "LICENSE"
 repository = "https://github.com/cargodog/polynomials"
 
 [dependencies]
-serde = { version = "1.0", features = ["derive"]}
-# unused - num-traits = { version = "0.2.14", default-features = false }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
Ah yes, I've been meaning to do this!
Resolves: https://github.com/cargodog/polynomials/issues/24